[WIP] Update README.md

imtin opened the pull request:
Updated the readme.md file with instructions on how to play the game for the players. This addresses issue #3 . Can you @franm574 please review the instructions and insure that they make sense?

imtin committed:
Update README.md

Updated the readme.md file with instructions on how to play the game.

franm574 commented on README.md:
```diff
@@ -1,17 +1,17 @@
+3. Use the left and right arrow keys to position the shape.
+4. The goal is to create complete rows with no empty spaces.
+5. When completed, the rows will disappear.
 
```
We should tell the user player how to pause the game
```suggestion
6. To pause the game just press the space bar again. 
```

franm574 commented on README.md:
```diff
@@ -1,17 +1,17 @@
+1. Press the space bar to begin.
+2. Use the up and down arrow keys to rotate the shape.
+3. Use the left and right arrow keys to position the shape.
+4. The goal is to create complete rows with no empty spaces.
```
I think it would be a good idea to move the goal of the game to the beginning of the instructions.

franm574 reviewed (commented):
Hi @imtin I have finished reviewing the instructions and recommended some changes for you to consider.

franm574 reviewed (changes requested):
Hi @imtin  I have finished reviewing the instructions and recommended some changes for you to consider.

imtin committed:
Update README.md

Co-authored-by: Fran Martin <188087962+franm574@users.noreply.github.com>

imtin committed:
Update README.md

Made the suggested changes to the instructions on how to play the game and clearly defined the goal of the game.

franm574 reviewed (approved):
The instructions look complete now. Thanks for the great work @imtin
